src/Sarif.Viewer.VisualStudio/ErrorList/SarifResultTableEntry.cs

```diff
@@ -22,14 +22,35 @@ namespace Microsoft.Sarif.Viewer.ErrorList
 {
     internal sealed class SarifResultTableEntry : ITableEntry, IDisposable
     {
+        internal const string SuppressionStateColumnName = "suppression";
+
         private bool isDisposed;
 
         private readonly Dictionary<string, object> columnKeyToContent = new Dictionary<string, object>(StringComparer.InvariantCulture);
 
-        public static readonly ReadOnlyCollection<string> SupportedColumns = new ReadOnlyCollection<string>(new[] {
+        // The set of columns we display depends on the contents of the displayed log file.
+        // Per https://github.com/microsoft/sarif-visualstudio-extension/issues/114, we do not
+        // display "suppressed" or "absent" issues by default.
+        //
+        // If we open a log file that contains a suppressed issue, we display the "Suppression
```

Comment:
>contains [](start = 38, length = 8)

Great comment here, as noted, co-locating some of this w/the actual implementation rendering this behavior is a good idea.

Reply:
Done.

---
In reply to: [538756999](https://github.com/microsoft/sarif-visualstudio-extension/pull/313#discussion_r538756999) [](ancestors = 538756999)